src/main/kotlin/apoc/nlp/NLPHelperFunctions.kt

```diff
@@ -0,0 +1,31 @@
+package apoc.nlp
+
+import apoc.util.Util
+import org.neo4j.graphdb.Node
+import org.neo4j.graphdb.Relationship
+import org.neo4j.graphdb.RelationshipType
+import org.neo4j.graphdb.Transaction
+import java.util.stream.Stream
+
+class NLPHelperFunctions {
+    companion object {
+        fun createRelationships(node: Node, nodes: MutableSet<Node>, relationshipType: RelationshipType) =
+                sequence {
```

Comment:
do we need this lazy evaluation here? or we can just do a `map` over `nodes`?

Reply:
oh yeh good point, dunno why I complicated it